config/default/kustomization.yaml

```
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: cnbi-system
+# namespace: cnbi-system
```

I am not sure this is going to work as intended.

There are several resources in the configuration that are namespaced. Including the creation of a namespace itself in [config/manager/manager.yaml](https://github.com/AICoE/meteor-operator/blob/e743c2848c468433592be1ba53c2fe8d4bdb96b5/config/manager/manager.yaml#L1-L6)

By removing this custom namespace, we are only letting the default namespace (`system`). But the problem/challenge that this causes will basically be the same, right?

---

I think we should not remove this configuration specific to the operator. Ultimately it will try to deploy to the `system` namespace (as I understood it). 

Why dont we create the `cnbi-system` namespace via one PR on op1st, and let let argocd create it, and then have kfdef changed to deploy meteor-operator to the cluster?

---

That could be a simple/straightforward option.

Moreover, as the `aicoe-meteor` namespace is [already known to operate first](https://github.com/operate-first/apps/tree/master/cluster-scope/base/core/namespaces/aicoe-meteor), we could switch back here to that namespace (i.e. go back to [what's in the main branch](https://github.com/AICoE/meteor-operator/blob/c3f30f2b75d37fce20b0d1390b91db084c7d319f/config/default/kustomization.yaml#L2)):

```suggestion
namespace: aicoe-meteor
```

and just [add that namespace to odh-cl1](https://github.com/operate-first/apps/pull/2398/commits/fd5572530dd789e25f6991a3569da6959896c735).